year-pages/2019-conferences-list.md

```diff
@@ -2,6 +2,8 @@
 
 | Conference | Date | Venue | Description | Scholarship |
 |------------|------|-------|-------------|-------------|
+| [ReactFoo](https://reactfoo.in/2019/) | 1-2 Mar | NIMHANS Convention Centre, Bangalore | India’s largest React conference for JavaScript and front-end engineers, cross-platform developers. | No |
```

Comment:
Would prefer the description to be `On React, alternatives to React, React Native and front-end engineering.` because that clearly states what this conference is about.